.github/workflows/wide_integer.yml

```diff
@@ -7,6 +7,76 @@ on:
   schedule:
     - cron: '0 2 * * *' # run at 2 AM UTC
 jobs:
+  cmake-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [ g++, clang++ ]
+        include:
+          - compiler: g++
+            container: johnmcfarlane/cnl_ci:gcc-11
```

Comment:
I have many more of these images. If you were interested in speeding up jobs by downloading ready-baked environments with compilers, Boost, and other necessities, we could consider using these images in the other jobs or uploading your own to Docker Hub.